dev/FHIR3-0-2-Geboortezorg/Zwangerschapskaart/_LoadResources/load-resources-purgecreateupdate-xml.xml

```diff
@@ -1089,7 +1089,7 @@
       <autocreate value="false"/>
       <autodelete value="false"/>
       <resource>
-         <reference value="../_reference/Vrouw1-Kaart2/medmij-gbz-bc-ObstetricProcedure236992007-d735d451-2062-11ec-2089-020000000000.xml"/>
+         <reference value="../_reference/Vrouw1-Kaart2/medmij-gbz-bc-ObstetricProcedure114961000146100-d735d451-2062-11ec-2089-020000000000.xml"/>
       </resource>
```

Comment:
Please add a display value for the reference.

Reply:
@NictizEduard dit is het load resources script, daar staan nergens display values bij ;)